### PR TITLE
Fix return value issue

### PR DESCRIPTION
**Pull request by daniel-weisse**
Use indirection to create ec2 client on demand

### CODE REVIEW

File: internal/attestation/aws/validator_test.go [hunk 1]
```diff
@@ -126,8 +128,8 @@ type AWSMetadataStub struct {
 // DescribeImages is a mock function for testing.
 // Although the original function works for multiple supplied image id's, this function ignores this, since in our code we also only get the information for one image.
 func (a *AWSMetadataStub) DescribeImages(ctx context.Context, params *ec2.DescribeImagesInput,
```
Comment (leongross) [inline internal/attestation/aws/validator_test.go:130]:
```suggestion
func (a *AWSMetadataStub) DescribeImages(ctx context.Context, params *ec2.DescribeImagesInput, optFns ...func(*ec2.Options),
) (*ec2.DescribeImagesOutput, error)
```
Maybe we should wrap up the function back into two lines to make it more readable?

**Review by leongross — APPROVED** [review]
LGTM

File: internal/attestation/aws/validator_test.go (hunk 1 above)
Comment (daniel-weisse) [reply internal/attestation/aws/validator_test.go:130]: This change was caused by my linter. 
If you function declaration is spread over multiple lines, the return values are always on their own separate line.

Comment (leongross) [reply internal/attestation/aws/validator_test.go:130]: Okay, than let's leave it like that.

**Pull request merged**